docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/home_landing/field--paragraph--cgov-column-one.html.twig

```diff
@@ -0,0 +1,6 @@
+{% if label %}
```

Comment:
Do you have an example of a use for label? otherwise I would say ignore it.

Reply:
I filled the column content fields with placeholder text and made them look nice. In actuality, if we're looking at Cancer Type Landing and News & Events Landing, then there isn't a case for labels. However, I don't see how or why we would render everything in those pages in a RAW HTML field. I say we just leave them as is until we're ready for the next chunk of work, setting those pages up.